source/numeric.d.ts

```diff
@@ -1,7 +1,51 @@
+import type {Not} from './internal';
+
 export type Numeric = number | bigint;
 
 type Zero = 0 | 0n;
 
+/**
+Returns the given number if it is a float, like `1.5` or `-1.5`.
```

Comment:
It doesn't return the number.

```suggestion
Returns whether the given number is a float, like `1.5` or `-1.5`.
```

Reply:
It should document the behavior for `Infinity`.